### ISSUE TITLE
current live CD fails for some locales

### ISSUE THREAD

**Issue by fossfreedom** [budgie-remix/budgie-remix-iso-builder#13]
The current pinguy method of building the ISO obviously doesnt deal with locales.

I've seen some launchpad bugs where ubiquity fails half way through the installation - I've managed to reproduce this very easily if you choose the Swedish locale (svenska) at the beginning to the installation and then install.

Lets test the new ISO builder to see if it deals with locales such as Swedish (svenska) correctly and allows the installation to complete correctly.

**Comment by uD4ra** [comment]
Yes, I have faced them when first time I'm using budgie-remix. Pinguy method is working fine only with Live version, If we jump directly to Install OS it won't working.
What about if  we can use the Live ISO customization method like [https://www.codinglogs.com/build-your-own-linux-live-distribution-based-on-ubuntu/](url) this ?

**Comment by fossfreedom** [comment]
fixed for yakkety with latest yakkety branch.

Need to backport for 16.04.2
